### DISCUSSION TITLE
A previous favorite feature required

### DISCUSSION THREAD

**Discussion by bethropolis** [dailydotdev/daily#500] [Feature Requests]
Previously on Daily dev you where able to filter search results depending on the source or tags, but after the new upgrade (the sidebar is nice by the way) it has become way way harder filtering through the results. Right now I have to visit the post page to find the url that will direct me to the tag results I want. Please help.

**Comment by nimrodkra** [comment]
Hi @bethropolis, thanks for the feedback! You're right, navigating to a feed of a specific tag or source isn't on the front page anymore. What I suggest is next time you visit a tag-specific or source-specific page, keep the URL in the custom shortcuts section :) Here's more info about it: https://app.daily.dev/posts/rIZ_uqa1Q and a [short video](https://twitter.com/dailydotdev/status/1498657307345010693?s=20&t=YwiGVlqCmhHhNH-0IvU7lg)

**Reply by bethropolis** [reply]
Thanks, this will do, let me set it up.

**Reply by bethropolis** [reply]
well @nimrodkra , the issue from using custom shortcuts is that the icons are the same and their is no other way of differentiating between multiple custom shortcuts.
![New-Tab](https://user-images.githubusercontent.com/66518866/158033582-3220962a-cdfb-4b15-9ab5-e526b28e08a0.png)

it's kind of hard going directly to the page you want, if you have multiple shortcuts. I recommend a feature where you can customize the icons to your own liking or a title field would help.

**Reply by nimrodkra** [reply]
Makes sense! We will consider this improvement. You got me curious with your original request, would you mind elaborating a little bit about how and why you use those tag-specific and source-specific feeds?

**Reply by bethropolis** [reply]
well the reason I used it was because It really enhanced my productivity because I spent less time looking for what I wanted rather than struggling to search for the specific source or tag.

How I used the tag-specific and source-specific feeds was that  I could select the tags and sources I wanted to filter my feed and directly choose any new article from my favourate sources. Right now with the feed filter it really doesn't make it easier and you can only filter tags.

**Reply by nimrodkra** [reply]
Got it. Makes sense. I'll take this feedback to our next product meeting and hopefully, we will be able to incorporate a solution for it down the roadmap. I can't commit to a specific time schedule at the moment though 🙏
